vcap/vcap/loading/errors.py

```diff
@@ -1,10 +1,21 @@
-class IncompatibleCapsuleError(Exception):
+class CapsuleError(Exception):
```

Comment:
Maybe consider calling this CapsuleLoadError, in case we decide to add a base CapsuleError in the future for _anything_ capsule related